includes/REST_API/Products_Controller.php

```diff
@@ -163,8 +163,15 @@ public function get_items( $request ) {
 		 *
 		 * @var string $search_term
 		 */
-		$search_term  = ! empty( $request['search'] ) ? $request['search'] : '';
-		$query_result = $query->get_search( $search_term );
+		$search_term = ! empty( $request['search'] ) ? $request['search'] : '';
+		
+		/**
+		 * Request context.
+		 *
+		 * @var string $sort_by
+		 */
+		$sort_by      = ! empty( $request['sort_by'] ) ? $request['sort_by'] : '';
```

Comment:
As per https://github.com/GoogleForCreators/web-stories-wp/pull/11552/files#r880339767, it becomes much easier with separate `order` and `orderby` params.

Also don't forget to declare new params in `get_collection_params()`

Something like 


```php
/**
 * Retrieves the query params for the products collection.
 *
 * @since 1.21.0
 *
 * @return array Collection parameters.
 */
public function get_collection_params(): array {
	$query_params = parent::get_collection_params();

	$query_params['order'] = [
		'description' => __( 'Order sort attribute ascending or descending.', 'web-stories' ),
		'type' => 'string',
		'default' => 'desc',
		'enum' => ['asc', 'desc'],
	];

	$query_params['orderby'] = [
		'description' => __( 'Sort collection by product attribute.', 'web-stories' ),
		'type' => 'string',
		'default' => 'date',
		'enum' => [
			'date',
			'price',
			'title',
		],
	];

	return $query_params;
}
```

Reply:
Massive plus one here. Looking at this, not is the cleaner to understand, but is inline with other endpoints.

Reply:
Added.